.changeset/mighty-hairs-train.md

```diff
@@ -0,0 +1,61 @@
+---
+"@comet/admin": minor
+---
+
+Add a more actions menu helper component
+
+This is done to provide a simple api to create a more actions menu for a list of items. 
+
+**Example:**
+
+```tsx
+<MoreActionsMenu
+    selectionSize={selectionSize}
+>
+    {({ handleClose }) => (
+        <>
+            <MenuItem
+                onClick={() => {
+                    handleMoveClick();
+                    handleClose();
+                }}
+            >
+                <ListItemIcon>
+                    <Move />
+                </ListItemIcon>
+                <ListItemText primary={"Move"} />
+            </MenuItem>
+
+            <MoreActionsDivider />
```

Comment:
Can't this be a normal MUI Divider?

Reply:
It is but abstracted to have a predefined styling, which can also be overwritten.

Reply:
I believe this styling should be for all dividers in menus. @jamesricky is this something we can achieve in the theme?

Reply:
I think dividers in menus should all look the same, is MUIs `Divider` not already styled correctly?
If not, it should be changed in the theme.